src/components/api-request.js

```diff
@@ -331,149 +341,112 @@ export default class ApiRequest extends LitElement {
           false,
         )[0].exampleValue;
       }
-      const labelColWidth = 'read focused'.includes(this.renderStyle) ? '200px' : '160px';
+      // const labelColWidth = 'read focused'.includes(this.renderStyle) ? '200px' : '160px';
```

Comment:
Maybe this line could be removed to avoid unnecessary comments

Reply:
I will change that